src/Host.cpp

```diff
@@ -523,34 +523,6 @@ void Host::sendRaw( QString command )
     mTelnet.sendData( command );
 }
 
-
-/*QStringList Host::getBufferTable( int from, int to )
```

Comment:
Buffers are still there, [there's code](https://github.com/svof/svof/blob/3a1ca4da34ec8a3f4f202b05cbf3e1fd0fb9a130/raw-svo.misc.lua#L154) that uses them - and they are very useful things to have. That said uninmplemented buffer methods can go.